diff --git a/Makefile.am b/Makefile.am
index 6d7085af..2fb56384 100755
--- a/Makefile.am
+++ b/Makefile.am
@@ -436,7 +436,6 @@ endif
 #
 test_apps = \
 	tests/test_ancillaries \
-	tests/test_system \
 	tests/test_pair_inproc \
 	tests/test_pair_tcp \
 	tests/test_reqrep_inproc \
@@ -885,13 +884,11 @@ endif
 if !ON_MINGW
 if !ON_CYGWIN
 test_apps += \
-	tests/test_shutdown_stress \
 	tests/test_ipc_wildcard \
 	tests/test_pair_ipc \
 	tests/test_rebind_ipc \
 	tests/test_reqrep_ipc \
 	tests/test_use_fd \
-	tests/test_zmq_poll_fd \
 	tests/test_timeo \
 	tests/test_filter_ipc
 
